packages/actor-query-operation-orderby-sparqlee/test/ActorQueryOperationOrderBySparqlee-test.ts

```diff
@@ -146,3 +146,117 @@ describe('ActorQueryOperationOrderBySparqlee', () => {
     });
   });
 });
+
+describe('ActorQueryOperationOrderBySparqleeMultipleComparators', () => {
```

Comment:
```suggestion
describe('ActorQueryOperationOrderBySparqlee with multiple comparators', () => {
```
To avoid confusion with class names.